bitnami/keycloak/21/debian-11/prebuildfs/opt/bitnami/scripts/libvalidations.sh

```diff
@@ -187,6 +187,28 @@ validate_port() {
     fi
 }
 
+########################
+# Validate if the provided argument is a valid IPv4 or IPv6 address
+# Arguments:
+#   $1 - IP to validate
+# Returns:
+#   Boolean
+#########################
+validate_ip() {
+    local ip="${1:?ip is missing}"
+    local stat=1
+
+    validate_ipv4 "$ip"
+    if [[ $? -eq 0 ]]; then
```

Comment:
```suggestion
    if validate_ipv4 "$ip"; then
```
https://www.shellcheck.net/wiki/SC2181